t/plugin/opentelemetry4-bugfix-pb-state.t

```diff
@@ -16,55 +16,34 @@
 #
```

Comment:
This file has been renamed so that opentelemetry.t test executes first. This is because this test file overrides/mocks a library function. The real tests need the real function and not mock function.